src/ILCompiler.WebAssembly/src/CodeGen/ILToWebAssemblyImporter.cs

```diff
@@ -3862,8 +3928,8 @@ void BuildAddOverflowChecksForSize(ref LLVMValueRef llvmCheckFunction, LLVMValue
                 if (signed)
                 {
                     builder.PositionAtEnd(elseBlock);
-                    //  b < int.MinValue - a
-                    BuildOverflowCheck(builder, leftOp, rightOp, minValue, LLVMIntPredicate.LLVMIntSLT, ovfBlock, noOvfBlock);
+                    // a - b overflows when b is -ve if  a > max + b
```

Comment:
What does `-ve` stand for?

Reply:
negative.  Maybe -ve is an electrical thing?  I'll expand it.